common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -211,78 +222,82 @@ public AwakenedProgressBarOverlay() {
         }
 
         @Override
-        public void render(PoseStack poseStack, float partialTicks, Window window) {
-            if (!WynnUtils.onWorld()) return;
+        public String icon() {
+            return "⚕";
+        }
 
-            WynnBossBarUtils.AwakenedBar awakenedBar = WynnBossBarUtils.getAwakenedBar();
-            if (awakenedBar.equals(WynnBossBarUtils.NO_AWAKENED_PROGRESS)) return;
+        @Override
+        public WynnBossBarUtils.BarProgress progress() {
+            return WynnBossBarUtils.getBloodPool(!INSTANCE.shouldDisplayOnBossBar);
```

Comment:
`CustomBarsOverlayFeature.this` instead of `INSTANCE`?

Reply:
This is an unhappy side effect that the overlay has to reference the config statically. I am glad for an alternative.